### ISSUE TITLE
Add Quit option to Buildasaur menu

### ISSUE THREAD

**Issue by voidrender** [czechboy0/Buildasaur#284]
It seems that the only way to quit Buildasaur at the moment is to kill the process. Should a Quit item be added to the Buildasaur menu?

EDIT: Re-added, as it seems it was there in the past--it's in the screenshots in the readme.

**Comment by czechboy0** [comment]
Yeah, I removed it as there was some weird issue, but if you have time to readd it, I'd accept it
